kotsu/model_registration.py

```diff
@@ -0,0 +1,10 @@
+"""Registration of models.
+
+Enables registering specifications of models and parameters combinations, under a unique ID,
+which can be passed to kotsu's run interface.
+"""
+from kotsu import _registration
+
+
+ModelSpec = _registration.EntitySpec
+ModelRegistry = _registration.EntityRegistry
```

Comment:
you _could_ just have this at the end of the `registration` file, so a user does like
```
from kotsu.registration import ModelRegistry, ValidationRegistry

model_registry = ModelRegistry()
...
```
might be a bit easier than having to import separately from `model_registration` and `validation_registration`

Reply:
you could even just expect users to make a
```
model_registry = Registry()
validation_registry = Registry()
```
but i guess you already considered that and prefer the explicitness

Reply:
> you _could_ just have this at the end of the `registration` file, so a user does like
> 
> ```
> from kotsu.registration import ModelRegistry, ValidationRegistry
> 
> model_registry = ModelRegistry()
> ...
> ```
> 
> might be a bit easier than having to import separately from `model_registration` and `validation_registration`

yea this is better, then I will `_...` the general classes.

Reply:
> you could even just expect users to make a
> 
> ```
> model_registry = Registry()
> validation_registry = Registry()
> ```
> 
> but i guess you already considered that and prefer the explicitness

Yea from a UX/discoverability point of view, I think better to have them explicitly look different at import time.